lib/Github/Api/PullRequest.php

```diff
@@ -156,7 +156,12 @@ public function create($username, $repository, array $params)
             throw new MissingArgumentException(['issue', 'body']);
         }
 
-        return $this->post('/repos/'.rawurlencode($username).'/'.rawurlencode($repository).'/pulls', $params);
+        $headers = [];
+        if (isset($params['draft']) && $params['draft'] == true) {
+            $headers['Accept'] = 'application/vnd.github.shadow-cat-preview+json';
```

Comment:
Maybe we should use the `acceptHeaderValue ` property of the`AcceptHeaderTrait`. This way we set custom headers like we done all over the codebase and it will make it easier to search for those usages and clean them up after the "beta" periods.

Reply:
Hey, I just realized that none of the functions which use the acceptHeaderValue from the trait have a test so I'm not sure how to assert that the trait has been used properly.

Reply:
Let's keep it like this for now, I will think of a way to test these preview endpoints in the future!